docker/README.md

```diff
@@ -0,0 +1,78 @@
+## Running `teos` in a docker container
+A `teos` image can be built from the Dockerfile located in `docker`. You can create the image by running:
+
+	cd rust-teos
+	docker build -f docker/Dockerfile -t teos .
+	
+Then you can create a container by running:
+
+	docker run -it -e <ENV_VARIABLES> teos
+	
+Notice that ENV variables are optional, if unset the corresponding default setting is used. The following ENVs are available:
+
+```
+- API_BIND=<teos_api_hostname>
+- API_PORT=<teos_api_port>
+- RPC_BIND=<teos_rpc_hostname>
+- RPC_PORT=<teos_rpc_port>
+- BTC_NETWORK=<btc_network>
+- BTC_RPC_CONNECT=<btc_node_hostname>
+- BTC_RPC_PORT=<btc_node_port>
+- BTC_RPC_USER=<btc_rpc_username>
+- BTC_RPC_PASSWORD=<btc_rpc_password>
+- BTC_RPC_CONNECT=<btc_zmq_hostname>
+- BTC_RPC_PORT=<btc_zmq_port>
+```
+
+If you are running `teos` and `bitcoind` in the same machine, continue reading for how to create the container based on your OS.
+
+### `bitcoind` running on the same machine (UNIX)
+The easiest way to run both together in he same machine using UNIX is to set the container to use the host network.
+	
+For example, if both `teos` and `bitcoind` are running on default settings, run
+    
+```
+docker run --network=host \
+  --name teos \
+  -e BTC_RPC_USER=<rpc username> \
+  -e BTC_RPC_PASSWORD=<rpc password> \
+  -it teos
```

Comment:
There was a section for that in the original file: https://github.com/talaia-labs/python-teos/blob/master/README.md

Reply:
Not sure if you saw my comment above but since the Dockerfile now automatically creates a volume, I removed these instructions. But happy to swap that out in favor of the volume instructions instead. I have no strong opinions either way.

Reply:
Docker creates a new volume each time the container is booted up from the image (unnamed volume) and the volume resides somewhere in `/var/lib/docker`.
With the `-v` it will let the user choose the host mount of the volume so it's reusable with later images.